nesh-dev/Store-Manager
# #161082040 host to github pages

### What does this PR do? 
Has the templates hosted on gh-pages 

### Description of Tasks 
Create an index.html to work as a landing page
Host the UI pages 

### How to Test this Manually?

- Clone this repo `git clone https://github.com/kevinene91/Store-Manager.git`
- Check out the correct branch `git fetch  origin gh-pages`
- Navigate to the repository folder and open the index.html
- Get the hosted project on [gh-pages](https://kevinene91.github.io/Store-Manager/index.html)

###  ScreenShot 
![hosted](https://user-images.githubusercontent.com/11676467/46940080-8325b080-d070-11e8-8a4a-f05eb91a8b1e.png)
